src/main/java/com/synopsys/integration/jira/common/rest/service/PluginManagerService.java

```diff
@@ -84,57 +85,74 @@ public JiraAppService(Gson gson, JiraHttpClient httpClient, JiraService jiraClou
         return Optional.empty();
     }
 
-    public InstalledAppsResponseModel getInstalledApps(String username, String accessToken) throws IntegrationException {
-        Request.Builder requestBuilder = createBasicRequestBuilder(getBaseUrl() + API_PATH, username, accessToken);
+    public InstalledAppsResponseModel getInstalledApps(String username, String accessTokenOrPassword) throws IntegrationException {
+        Request.Builder requestBuilder = createBasicRequestBuilder(createBaseRequestUrl(), username, accessTokenOrPassword);
         requestBuilder.addQueryParameter(QUERY_KEY_OS_AUTH_TYPE, QUERY_VALUE_OS_AUTH_TYPE);
         requestBuilder.method(HttpMethod.GET);
         requestBuilder.addAdditionalHeader("Accept", MEDIA_TYPE_INSTALLED);
 
-        return jiraCloudService.get(requestBuilder.build(), InstalledAppsResponseModel.class);
+        return jiraService.get(requestBuilder.build(), InstalledAppsResponseModel.class);
     }
 
     public Response installMarketplaceApp(String addonKey, String username, String accessToken) throws IntegrationException {
```

Comment:
Should we make this cloud-specific?

Reply:
I've gone back and forth on this. I had a cloud app service and server service.  The only methods that are different are the installation methods.  All the other methods in the PluginManagerService works for both.